CHANGES.rst

```diff
@@ -6,6 +6,60 @@ https://hyperspy.readthedocs.io/en/latest/user_guide/changes.html
 
 .. towncrier-draft-entries:: |release| [UNRELEASED]
 
+
+API Removal
+===========
```

Comment:
Shouldn't this be a heading of one level lower than the new release?
(Also applies to the other deprecation PRs)

Reply:
It doesn't play well with towncrier, so I would suggest to leave it as it is and we edit it during when we do the release.

Reply:
Yep, just realized that as well.